### PR TITLE
#4332 [Manager]Incorrect node items

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4332 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyView/PipelinePropertyView.js [hunk 1]
```diff
@@ -251,25 +251,28 @@ const PipelinePropertyView = props => {
...
-                      component="span"
-                    >
-                      {state}
-                    </Typography>
-                  }
-                />
-              );
-            })}
+            {tasksStatus
+              // UI now not support show master node.
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyView/PipelinePropertyView.js]: What about?

```suggestion
              // Master node is not displaying in the UI
```

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/PipelinePropertyView.js]: Is master plural? Then we should change this to _Master nodes are..._

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
@chia7712  This pr is ready to merge, thanks

**Pull request merged**